test/support/page_objects.js

```diff
@@ -0,0 +1,74 @@
+
+import Common from './pages/common.js';
+import ConsolePage from './pages/console_page.js';
+import DashboardPage from './pages/dashboard_page.js';
+import DiscoverPage from './pages/discover_page.js';
+import HeaderPage from './pages/header_page.js';
+import SettingsPage from './pages/settings_page.js';
+import ShieldPage from './pages/shield_page.js';
+import VisualizePage from './pages/visualize_page.js';
+
+const common = new Common();
+const consolePage = new ConsolePage();
+const dashboardPage = new DashboardPage();
+const discoverPage = new DiscoverPage();
+const headerPage = new HeaderPage();
+const settingsPage = new SettingsPage();
+const shieldPage = new ShieldPage();
+const visualizePage = new VisualizePage();
+
+export default {
```

Comment:
Oh, the only thing I'm asking is that we change it from a plain object to an instance of a class 😄 

Something like:
```js
class PageObjects {
  remote = null;
  init(remote) {
    this.remote = remote
  }
}

export default new PageObjects
```

I don't want to jump through those whoops either, but if we are going to be tracking state with `this` rather than the shared scope I think we should signify that intent by using a `class`.